dual_net.py

```diff
@@ -206,8 +206,8 @@ def my_res_layer(inputs):
     # train ops
     global_step = tf.train.get_or_create_global_step()
     policy_cost = tf.reduce_mean(
-        tf.nn.softmax_cross_entropy_with_logits(
-            logits=logits, labels=labels['pi_tensor']))
+        tf.nn.softmax_cross_entropy_with_logits_v2(
+            logits=logits, labels=tf.stop_gradient(labels['pi_tensor'])))
```

Comment:
curious why this is necessary

Reply:
Ah, explained this on the other PR https://github.com/tensorflow/minigo/pull/139#discussion_r177646634 but forgot to copy it over:

This `tf.stop_gradient` may not be necessary, but I added it just to be safe.

See https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits and https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits_v2 for context.